awscli/customizations/s3/subcommands.py

```diff
@@ -266,12 +266,20 @@ def get_endpoint(service, region, endpoint_url, verify):
                                 verify=verify)
 
 
+def get_client(session, region, endpoint_url, verify):
```

Comment:
Why is this a module function?  Given the s3 commands will be using this, it seems to make the most sense as an internal method.